File: 263.Ugly Number.py
class Solution(object):
    def isUgly(self, num):
        """
        :type num: int
        :rtype: bool
        """
        if num<=0:
            return False
        for d in (2,3,5):
            while num/d*d==num:
                num=num/d
        if num==1:
            return True
        return False
        

File: 263.Ugly Number.c
bool isUgly(int num){
    if (num < 1) return false;
    int factors[] = {2, 3, 5};
    for (int i = 0; i < 3; i++) {
        while (!(num % factors[i])) num /= factors[i];
    }
    return num == 1;
}
